src/components/Combobox/Dynamic/DynamicCombobox.tsx

```diff
@@ -142,7 +144,12 @@ const DynamicComboboxInner = <T extends Option>(
       <Portal asChild ref={refs.setFloating} style={floatingStyles}>
         <Box
           position="relative"
-          display={getListDisplayMode({ isOpen, hasItemsToSelect, loading })}
+          display={getListDisplayMode({
+            isOpen,
+            loading,
+            hasItemsToSelect,
+            showEmptyState: !!locale?.noItems,
```

Comment:
**Suggestion:** I'm wondering if  we should tie display mode with user passed locale? Shouldn't it be based on `hasItemsToSelect` instead?